### DISCUSSION TITLE
What is GitHub?

### DISCUSSION THREAD

**Discussion by montasim** [STRINGLABITSOLUTIONS/Bhojon#234] [Q&A] [answered]
Short answer.

**Comment by fcasamento** [comment, marked as answer]
A solution previously created by Chris Wanstrath, J. Hyett, Tom Preston-Werner and Scott Chacon and in which in 2018 it was purchased by Microsoft. Its role started as a "competitor" of GitLab and today it has many features.

**Reply by montasim** [reply]
@fcasamento Now you have unlocked the Galaxy Brain badge.

**Comment by d4tdev** [comment]
At a high level, GitHub is a website and cloud-based service that helps developers store and manage their code, as well as track and control changes to their code. To understand exactly what GitHub is, you need to know two connected principles:

+ Version control
+ Git

In this article, we’ll first explain those two principles. Then, we’ll dig into more about GitHub and how you can actually use GitHub to work with WordPress and/or Kinsta.

**Reply by montasim** [reply]
@d4rtj try answering the questions that are not answered yet. You can find them form here: https://github.com/STRINGLABITSOLUTIONS/Bhojon/discussions?discussions_q=is%3Aunanswered
